apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -81,7 +87,7 @@ function BaseTemplateEditorPage() {
           </ReactFlowProvider>
         </form>
       </PageContainer>
-      <BlueprintModal />
+      {shouldRenderBlueprintModal && <BlueprintModal />}
```

Comment:
Do not render the blueprint modal and don't call query hooks when there is no `blueprintId` set.

I've noticed this issue in the tests, like when env is switched we do call refetch all queries, and blueprint queries do fail which results in that the page is navigated from the editor to the workflows, causing tests to fail.